### PR TITLE
file added

### PR DESCRIPTION
**Pull request by vivek1214-git**
file added

### CODE REVIEW

File: app/Http/Controllers/DataTest.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    public function store(Request $request)
+    {
+        $request->validate([
+            // 'email' => 'required|email',
+            // 'phone' => 'required|size:10|digits:10',
+            // 'address' => 'required|min:10|max:100',
+         //  'testtype' => 'required',
+            'session_date' => 'required',
+             'session_time' => 'required',
+             'meeting_name' => 'required',
```
Comment (kkaajjaall) [inline app/Http/Controllers/DataTest.php:21]: remove the extra space

File: app/Http/Controllers/DataTest.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+use Illuminate\Http\Request;
+use App\Http\Controllers\DataTest;
+
+class DataTest extends Controller
+{
+
+    public function store(Request $request)
+    {
+        $request->validate([
+            // 'email' => 'required|email',
```
Comment (kkaajjaall) [inline app/Http/Controllers/DataTest.php:15]: remove the comment lines

File: app/Http/Controllers/Selfie.php [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+use Illuminate\Http\Request;
+
+class Selfie extends Controller
+{
+
+    public function store(Request $request)
+    {
+        $request->validate([
+            // 'email' => 'required|email',
+            // 'phone' => 'required|size:10|digits:10',
```
Comment (kkaajjaall) [inline app/Http/Controllers/Selfie.php:14]: remove comment lines

File: app/Http/Controllers/Selfie.php [hunk 4]
```diff
@@ -4,9 +4,7 @@
```
Comment (kkaajjaall) [inline app/Http/Controllers/Selfie.php:4]: change your file name
Selfie->SelfieController

File: app/Http/Controllers/TestData.php [hunk 5]
```diff
@@ -20,6 +21,13 @@ public function store(Request $request)
...
+        ExamForm::create([
+            'exam_date' => $request->get ('exam_date'),
+            'score' => $request->get ('score')
+
+          ]);
+          return view('success');
+
+
     }
 }
```
Comment (kkaajjaall) [inline app/Http/Controllers/TestData.php:33]: change file name
TestData->ExtraScoredController

File: app/Models/ExamForm.php [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class ExamForm extends Model
+{
+    use HasFactory;
+    protected $table = "examform";
+    protected $guarded = [];
+}
```
Comment (kkaajjaall) [inline app/Models/ExamForm.php:13]: change model name
ExamForm->ExtraScored

File: app/Models/Form.php [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Form extends Model
+{
+    use HasFactory;
+    protected $table = "formdata";
+    protected $guarded = [];
+}
```
Comment (kkaajjaall) [inline app/Models/Form.php:13]: change model name
Form->CounselorSession

File: resources/views/counselorsession.blade.php [hunk 8]
```diff
@@ -4,49 +4,61 @@
     <div class="container">
         <h3>Apply to session</h3>
 
-        <form class="row g-3">
+        <form class="row g-3" method="post" action="/test2">
+            @csrf
             <div class="col-md-6">
                 <label for="inputDate" class="form-label">Date</label>
-                <input type="date" class="form-control" name="session_date" id="inputDate">
+                <input type="date" class="form-control @error('session_date') is-invalid @enderror" name="session_date" id="inputDate"  value="{{ old('session_date') }}">
```
Comment (kkaajjaall) [inline resources/views/counselorsession.blade.php:11]: all id should be meaning full

File: routes/web.php [hunk 9]
```diff
@@ -35,3 +36,5 @@
 });
 
 Route::post('/test', [TestData::class, 'store']);
+Route::post('/test2', [DataTest::class, 'store']);
+Route::post('/test3', [Selfie::class, 'store']);
```
Comment (kkaajjaall) [inline routes/web.php:40]: format all the file and remove extra space

File: app/Http/Controllers/CounselorSessionController.php [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+{
+
+    public function store(Request $request)
+    {
+        $request->validate([
+            'session_date' => 'required',
+            'session_time' => 'required',
+            'meeting_name' => 'required',
+        ]);
+        CounselorSessionModel::create([
```
Comment (ithqindia) [inline app/Http/Controllers/CounselorSessionController.php:20]: This should be CounselorSession

File: app/Models/CounselorSessionModel.php [hunk 11]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class CounselorSessionModel extends Model
```
Comment (ithqindia) [inline app/Models/CounselorSessionModel.php:8]: CounselorSession

File: app/Models/CounselorSessionModel.php [hunk 12]
```diff
@@ -0,0 +1,13 @@
...
+{
+    use HasFactory;
+    protected $table = "counselorsessions";
```
Comment (ithqindia) [inline app/Models/CounselorSessionModel.php:11]: counselor_sessions

File: app/Models/ExtraScore.php [hunk 13]
```diff
@@ -0,0 +1,13 @@
...
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+
+class ExtraScore extends Model
+{
+    use HasFactory;
+    protected $table = "extrascores";
```
Comment (ithqindia) [inline app/Models/ExtraScore.php:11]: extra_scores

File: database/migrations/2021_12_15_182433_create_formdata.php [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+class CreateFormdata extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('counselorsessions', function (Blueprint $table) {
```
Comment (ithqindia) [inline database/migrations/2021_12_15_182433_create_formdata.php:16]: counselors_sessions

File: database/migrations/2021_12_15_191651_create_examform.php [hunk 15]
```diff
@@ -0,0 +1,33 @@
...
+class CreateExamform extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('extrascores', function (Blueprint $table) {
```
Comment (ithqindia) [inline database/migrations/2021_12_15_191651_create_examform.php:16]: exra_scores

File: database/migrations/2021_12_16_080643_add_user_id.php [hunk 16]
```diff
@@ -0,0 +1,44 @@
...
+    public function up()
+    {
+        Schema::table('counselorsessions', function ($table) {
+            $table->unsignedBigInteger('user_id');
+        });
+        Schema::table('extrascores', function ($table) {
+            $table->unsignedBigInteger('user_id');
+        });
+        Schema::table('selfies', function ($table) {
+            $table->unsignedBigInteger('user_id');
```
Comment (ithqindia) [inline database/migrations/2021_12_16_080643_add_user_id.php:23]: Changes these too

File: dropColumn('user_id') [hunk 17]
```diff
@@ -0,0 +1,5 @@
+
+                                                                              
+  Too many arguments to "make:migration" command, expected arguments "name".
```
Comment (ithqindia) [inline dropColumn('user_id'):3]: Delete this file

File: resources/views/counselorsession.blade.php [hunk 18]
```diff
@@ -0,0 +1,102 @@
+@extends('layouts.main')
+@section('main-content')
+
+    <div class="container">
+        <h3>Apply to session</h3>
+        <form class="row g-3" method="post" action="/test2">
+            {{-- error code takjoooooooooooooooooooo --}}
```
Comment (ithqindia) [inline resources/views/counselorsession.blade.php:7]: Remove this comment

File: resources/views/counselorsession.blade.php [hunk 19]
```diff
@@ -0,0 +1,102 @@
...
+        </form>
+    </div>
+    <!-- Option 1: Bootstrap Bundle with Popper -->
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"
+        integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous">
+    </script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script>
+        $("[name='meeting_name']").click(function(event) {
+            if ($('#input_Radio').is(":checked")) {
```
Comment (ithqindia) [inline resources/views/counselorsession.blade.php:95]: input_Radio change to something meaningful

File: resources/views/extrascore.blade.php [hunk 20]
```diff
@@ -0,0 +1,52 @@
+@extends('layouts.main')
+@section('main-content')
+    <div class="container">
+        <h3>English and standardised test scores</h3>
+        <form class="row row-cols-lg-auto g-3 align-items-center" method="post" action="/test">
```
Comment (ithqindia) [inline resources/views/extrascore.blade.php:5]: change action to meaningful

File: resources/views/extrascore.blade.php [hunk 21]
```diff
@@ -0,0 +1,52 @@
...
+                <select name="user_id">
+                    @foreach ($users as $user)
+                        <option value="{{ $user->id }}">{{ $user->name }} {{ $user->email }}</option>
+                    @endforeach
+                </select>
+            </div>
+            <fieldset>
+                <div class="mb-3">
+                    <label for="Select" class="form-label">Test type</label>
+                    {{-- value =1 kadjooooo ani barobar nav takjoooooooooooooooooo --}}
```
Comment (ithqindia) [inline resources/views/extrascore.blade.php:17]: remove comment

File: routes/web.php [hunk 22]
```diff
@@ -1,22 +1,47 @@
...
     return view('welcome');
 });
-
 Route::get('/test', function () {
     return view('test');
 });
+Route::get('/selfie', function () {
+    return view('selfie');
+});
+Route::get('/counselorsession', function () {
```
Comment (ithqindia) [inline routes/web.php:27]: counselor-session

File: routes/web.php [hunk 23]
```diff
@@ -1,22 +1,47 @@
...
+    return view('counselorsession');
```
Comment (ithqindia) [inline routes/web.php:28]: counselor-session

File: routes/web.php [hunk 24]
```diff
@@ -1,22 +1,47 @@
...
+});
+Route::get('/extrascore', function () {
```
Comment (ithqindia) [inline routes/web.php:30]: extra-scores

File: routes/web.php [hunk 25]
```diff
@@ -1,22 +1,47 @@
...
+    return view('extrascore');
```
Comment (ithqindia) [inline routes/web.php:31]: extra-scores

File: routes/web.php [hunk 26]
```diff
@@ -1,22 +1,47 @@
...
+});
+Route::get('/counselorsession', function () {
```
Comment (ithqindia) [inline routes/web.php:33]: counselor-session

File: routes/web.php [hunk 27]
```diff
@@ -1,22 +1,47 @@
...
+    $users = User::get()->sortBy('name');
+    return view('counselorsession', compact('users'));
```
Comment (ithqindia) [inline routes/web.php:35]: counselor-session

File: routes/web.php [hunk 28]
```diff
@@ -1,22 +1,47 @@
...
+});
+Route::get('/extrascore', function () {
```
Comment (ithqindia) [inline routes/web.php:37]: extra-score

File: routes/web.php [hunk 29]
```diff
@@ -1,22 +1,47 @@
...
+    $users = User::get()->sortBy('name');
+    return view('extrascore', compact('users'));
```
Comment (ithqindia) [inline routes/web.php:39]: extra-score

File: routes/web.php [hunk 30]
```diff
@@ -1,22 +1,47 @@
...
+});
+Route::get('/selfie', function () {
+    $users = User::get()->sortBy('name');
+    return view('selfie', compact('users'));
+});
+Route::post('/test', [ExtraScoreController::class, 'store']);
+Route::post('/test2', [CounselorSessionController::class, 'store']);
+Route::post('/test3', [SelfieController::class, 'store']);
```
Comment (ithqindia) [inline routes/web.php:47]: change routes names to meanigful

File: app/Http/Controllers/CounselorSessionController.php [hunk 31]
```diff
@@ -43,6 +42,7 @@ public function update(Request $request)
 
         $userInformation->session_date = $request->get('session_date');
         $userInformation->session_time = $request->get('session_time');
+        $userInformation->session_time = $request->get('meeting_name');
```
Comment (amolswnz) [inline app/Http/Controllers/CounselorSessionController.php:45]: This should be 
`$userInformation->meeting_name = $request->get('meeting_name');`

File: app/Models/CounselorSession.php [hunk 32]
```diff
@@ -8,6 +8,6 @@
 class CounselorSession extends Model
 {
     use HasFactory;
-    protected $table = "counselor_sessions";
+    protected $table = "counselors";
```
Comment (ithqindia) [inline app/Models/CounselorSession.php:11]: protected $table = "counselor_sessions";
why you changed this

File: database/migrations/2021_12_17_110515_create_counselor_sessions.php [hunk 33]
```diff
@@ -13,8 +13,9 @@ class CreateCounseloursession extends Migration
      */
     public function up()
     {
-        Schema::create('counselor_sessions', function (Blueprint $table) {
+        Schema::create('counselors', function (Blueprint $table) {
```
Comment (ithqindia) [inline database/migrations/2021_12_17_110515_create_counselor_sessions.php:16]: Schema::create('counselor_sessions', function (Blueprint $table) {
change this

File: database/migrations/2021_12_17_110515_create_counselor_sessions.php [hunk 34]
```diff
@@ -29,6 +30,6 @@ public function up()
      */
     public function down()
     {
-        Schema::dropIfExists('counselor_sessions');
+        Schema::dropIfExists('counselors');
```
Comment (ithqindia) [inline database/migrations/2021_12_17_110515_create_counselor_sessions.php:33]: Schema::dropIfExists('counselor_sessions');
change this

File: database/migrations/2021_12_17_112501_add_user_id.php [hunk 35]
```diff
@@ -13,7 +13,7 @@ class AddUserId extends Migration
      */
     public function up()
     {
-        Schema::table('counselor_sessions', function ($table) {
+        Schema::table('counselors', function ($table) {
```
Comment (ithqindia) [inline database/migrations/2021_12_17_112501_add_user_id.php:16]: Schema::table('counselor_sessions', function ($table) {
change

File: resources/views/counselor-session.blade.php [hunk 36]
```diff
@@ -42,7 +46,7 @@
             <legend class="col-form-label col-sm-2 pt-0">Session Type</legend>
             <div class="col-sm-10">
                 <div class="form-check">
-                    <input class="form-check-input" type="radio" name="meeting_name" id="online_Radio" value="online">
+                    <input class="form-check-input" type="radio" name="meeting_name" id="gridRadios" value="online">
```
Comment (ithqindia) [inline resources/views/counselor-session.blade.php:49]: meaningful name

File: resources/views/counselor-session.blade.php [hunk 37]
```diff
@@ -78,7 +82,7 @@
                     </div>
                 </div>
                 <div class="form-check">
-                    <input class="form-check-input" type="radio" name="meeting_name" id="offline_Radio" value="offline">
+                    <input class="form-check-input" type="radio" name="meeting_name" id="gridRadio" value="offline">
```
Comment (ithqindia) [inline resources/views/counselor-session.blade.php:85]: meaningful name

File: resources/views/counselor-session.blade.php [hunk 38]
```diff
@@ -97,7 +101,7 @@
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
 <script>
     $("[name='meeting_name']").click(function(event) {
-        if ($('#input_Radio').is(":checked")) {
+        if ($('#gridRadios').is(":checked")) {
```
Comment (ithqindia) [inline resources/views/counselor-session.blade.php:104]: meaningful name

**Review by ithqindia — CHANGES_REQUESTED** [review]

**Review by ithqindia — APPROVED** [review]

**Review by ithqindia — APPROVED** [review]
Nice job

File: resources/views/counselor-session-info.blade.php [hunk 39]
```diff
@@ -1,14 +1,94 @@
...
+                </div>
+                <div class="row mb-7">
+                    <lable class="col-lg-4 fw-bold text-muted">Time </lable>
+                    <div class="col-lg-8">
+                        <span
+                            class="fw-bolder fs-6 text-gray-800">{{ $user->counselorsession_information->session_time }}</span>
+                    </div>
+                </div>
+                <div class="row mb-7">
+                    <lable class="col-lg-4 fw-bold text-muted">Session Type</lable>
```
Comment (manii283) [inline resources/views/counselor-session-info.blade.php:78]: change 2nd word in lower case

File: resources/views/counselor-session-info.blade.php [hunk 40]
```diff
@@ -1,14 +1,94 @@
...
+                <div class="row mb-7">
+                @else
+                    <h1>No information available</h1>
+    @endif
+    </div>
+    </div>
+    </div>
+    {{-- @php dd($user->toArray()) @endphp --}}
+    </div>
+@endsection
```
Comment (manii283) [inline resources/views/counselor-session-info.blade.php:94]: format the code

File: resources/views/counselor-session.blade.php [hunk 41]
```diff
@@ -45,7 +45,7 @@
         <fieldset class="row mb-3">
             <legend class="col-form-label col-sm-2 pt-0">Session Type</legend>
```
Comment (manii283) [inline resources/views/counselor-session.blade.php:46]: change the 2nd word in lower case

File: resources/views/extra-score-info.blade.php [hunk 42]
```diff
@@ -1,12 +1,87 @@
...
+    @if (isset($user->extrascore_information))
+        <div class=" card mb-5 mb-xl-10" id="kt_profile_details_view">
+            <div class="card-header cursor-pointer">
+                <div class="card-title m-0">
+                    <h3 class="fw-bolder m-0">ExtraScore User Information</h3>
+                </div>
+            </div>
+            <div class="card-body p-9">
+                <div class="row mb-7">
+                    <lable class="col-lg-4 fw-bold text-muted">Exam Date </lable>
```
Comment (manii283) [inline resources/views/extra-score-info.blade.php:65]: change 2nd word in lower case

File: resources/views/extra-score-info.blade.php [hunk 43]
```diff
@@ -1,12 +1,87 @@
...
+                @else
+                    <h1>No information available</h1>
+    @endif
+    </div>
+    </div>
+    </div>
+    {{-- @php dd($user->toArray()) @endphp --}}
+    </div>
+
+@endsection
```
Comment (manii283) [inline resources/views/extra-score-info.blade.php:87]: format code code

File: resources/views/selfie-info.blade.php [hunk 44]
```diff
@@ -1,11 +1,82 @@
...
+                <div class="row mb-7">
+                    <lable class="col-lg-4 fw-bold text-muted">Selfie </lable>
+                    <div class="col-lg-8">
+                        <span class="fw-bolder fs-6 text-gray-800">{{ $user->selfie_information->selfie }}
+                        </span>
+                    </div>
+                </div>
+
+
+                <div class="row mb-7">
```
Comment (manii283) [inline resources/views/selfie-info.blade.php:73]: remove extra space

File: resources/views/selfie.blade.php [hunk 45]
```diff
@@ -22,32 +22,32 @@
...
+                    @endisset
+            </select>
+        </div>
+        <div class="form-group">
+            <label for="selfie">Selfie</label>
+            <input type="file" class="form-control" name="selfie" id="selfie1" aria-describedby="SelfieHelp">
+            <small id="selfieHelp" class="form-text text-muted">PDF,JPG or PNG format and please do not exceed
+                5MB</small>
+        </div>
+        <button type="submit" class="btn btn-primary">Upload Image</button>
```
Comment (manii283) [inline resources/views/selfie.blade.php:50]: change the 2nd word in lower case

**Pull request closed without merging**